config/settings.yml

```diff
@@ -1686,3 +1686,12 @@ brd:
 
 xlsx_file_fetcher:
   github_access_token: ~
+
+vye:
+  ivr_key: '0615324c-5a8d-9fe1-3542-7b3b86df281f'
+  scrypt:
+    salt: '51463d6f-3b19-ae87-578a-7c4e3f1a5588'
+    N: 16384
+    r: 8
+    p: 1
```

Comment:
What do `N`, `r`, and `p` stand for here? If possible, can you spell them out? When variables are spelled out, the code is easier to search and is clearer to read.

Reply:
updated, please review

Reply:
Looks good. Thanks.